### PR TITLE
Add support for Network Load Balancer resources management

### PR DESCRIPTION
**Pull request by falzm**
This change implements support for Network Load Balancer resources
management.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ linters-settings:
 
 linters:
   enable:
-    - goconst
```
Comment (pierre-emmanuelJ) [inline .golangci.yml:7]: not possible with `// nolint:goconst` ?

Comment (falzm) [reply .golangci.yml:7]: I don't believe this check is relevant, to be honest.

File: cmd/nlb_list.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+	out := make(nlbListOutput, 0)
+	res := make(chan nlbListItemOutput)
+	defer close(res)
+
+	go func() {
+		for nlb := range res {
+			out = append(out, nlb)
+		}
+	}()
```
Comment (pierre-emmanuelJ) [inline cmd/nlb_list.go:61]: IMO this go routine is not useful in this case, can be appended directly in the result loop.

File: cmd/nlb_service_show.go [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+func (o *nlbServiceShowOutput) toJSON() { outputJSON(o) }
+func (o *nlbServiceShowOutput) toText() { outputText(o) }
+func (o *nlbServiceShowOutput) toTable() {
+	t := table.NewTable(os.Stdout)
+	defer t.Render()
+
+	t.SetHeader([]string{"NLB Service"})
+	t.Append([]string{"ID", o.ID})
```
Comment (pierre-emmanuelJ) [inline cmd/nlb_service_show.go:52]: `output()` dedicated function can not be used in this case?

File: cmd/nlb_list.go (hunk 2 above)
Comment (falzm) [reply cmd/nlb_list.go:61]: Tempting, but after a quick search it looks like it's not a good idea to append to a slice from multiple goroutines:

https://groups.google.com/g/golang-nuts/c/SRtSTar853g?pli=1
https://medium.com/@cep21/gos-append-is-not-always-thread-safe-a3034db7975
http://dnaeon.github.io/concurrent-maps-and-slices-in-go/

Alternatively, we could wrap the slice in a new type that includes a `sync.Mutex` for locking, but isn't it overkill?

Comment (pierre-emmanuelJ) [reply cmd/nlb_list.go:61]: After looking more deeply, didn't see your goroutine in `forEachZone `. Well let's keep like that 😉

Comment (pierre-emmanuelJ) [reply cmd/nlb_list.go:61]: >Alternatively, we could wrap the slice in a new type that includes a sync.Mutex for locking, but isn't it overkill?

Yes let's keep the go routine with the chan in this case

**Pull request merged**